### ISSUE TITLE
OTLP Tracing Endpoint Expects http:// Prefix

### ISSUE THREAD

**Issue by lleadbet** [apollographql/router#1754]
**Describe the bug**
As title; the OTLP tracing endpoint expects the `http://` prefix even if using gRPC.

**To Reproduce**
A non-working config: 

```yml
server:
  listen: 0.0.0.0:80
telemetry:
  tracing:
    otlp: 
      endpoint: collector.monitoring:4317
      protocol: grpc
    trace_config:
      service_name: "router"
      service_namespace: "router"
  metrics:
    prometheus:
      enabled: true
plugins:
  experimental.include_subgraph_errors:
    all: true
```

Returns either timeout errors or generic gRPC errors; example: `OpenTelemetry trace error occurred: Exporter otlp encountered the following error(s): the grpc server returns error (Unknown error): , detailed error message: transport error`

Working configuration: 

```yml
server:
  listen: 0.0.0.0:80
telemetry:
  tracing:
    otlp: 
      endpoint: http://collector.monitoring:4317
      protocol: grpc
    trace_config:
      service_name: "router"
      service_namespace: "router"
  metrics:
    prometheus:
      enabled: true
plugins:
  experimental.include_subgraph_errors:
    all: true
```

**Expected behavior**
Either: 

* Documented format for the URL 
* Accepting either format (prefixed and non-prefixed)

**Comment by abernix** [comment]
Can you include information about what version of the Router are you using?

**Comment by lleadbet** [comment]
I was running v0.16.0. 

This seems to have been resolved between versions 0.16 and 1.0-alpha-3, so closing the issue here.

**Comment by abernix** [comment]
Lovely! Thanks for reporting back!
